app/views/exhibit_filters/_edit.html.erb

```diff
@@ -0,0 +1,9 @@
+<div role="tabpanel" class="tab-pane" id="filter">
+
+<%= bootstrap_form_for [@exhibit, @exhibit.exhibit_filters.first_or_initialize] do |f| %>
```

Comment:
I've asked @ggeisler to give some thought to the UI for this behavior. I don't think it should necessarily hold up the PR, though.